app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(comment_params)
+    @review = Review.find(params[:review_id])
+    @dinosaur = @review.dinosaur
```

Comment:
I'd recommend using `.merge` in your `comment_params` to add the dinosaur and review information to a new comment. Then you can thin out this controller, which as it stands is a little long.